tests/lua-tests/src/controller.lua

```diff
@@ -1,18 +1,22 @@
-
+if (cc.PLATFORM_OS_OHOS == currPlatform) then
+local jit = require("jit")
+jit.off()
+end
```

Comment:
Why need to disable jit? And only disable jit in this test case?

Reply:
JIT shutdown is to prevent malicious software from abusing JIT memory to execute arbitrary code, bypass application market audits, and achieve the goal of changing faces